### ISSUE TITLE
LaTeX Interface

### ISSUE THREAD

**Issue by mforbes** [sagemathinc/cocalc-docker#192]
As we start writing some collaborative grants, I have a bunch of new users building LaTeX documents so am opening an issue to discuss UI issues we discover - especially from new users. *(Let me know if we should move this to a discussion.)*  For now I am just recording issues: we will circle back with suggestions if we have them later.

1. There is not a consistent and obvious "Build" button, especially if one's browser window is small.  For users coming from Overleaf, this might be confusing because they expect the PDF to be out of date.  Related confusions:
   1. The build button appears and disappears depending on which window has focus.
   2. The PDF previewer has no Build button.

   This is generally not an issue when people edit files online, but has been an issue when I edit a style file or similar offline (Emacs + TRAMP over SSH).

2. Said above, but deserves its own point.  The way buttons and functionality is hidden in smaller windows can be very confusing.  Perhaps when buttons are hidden, a "..." or hamburger could appear that could be clicked to give a full menu of options?

**Comment by williamstein** [comment]
I totally agree -- I would be open to just not hiding the buttons in a nonfocused window at all, and instead maybe just dimming them.  We now have a hamburger icon to show all buttons that aren't visible, so that helps reduce clutter, etc.

I'll try experimenting with not hiding the buttons in the non-focused windows this week.

**Comment by williamstein** [comment]
> Let me know if we should move this to a discussion.

This should be in the https://github.com/sagemathinc/cocalc issues, I think...
